cypress/integration/updates/sidebar.spec.js

```diff
@@ -76,7 +76,7 @@ describe("update notification", () => {
     });
 
     cy.visit("/");
-    cy.contains("Learn more").should("exist");
+    cy.contains("Learn more", { timeout: 60000 }).should("exist");
```

Comment:
Do you know why this failed now and not before ?

Reply:
It was a bit flaky, I've had three fails in a row, so I increased to to 20 seconds and it still failed. I'm surprised one minute worked. I'm not sure why it fails, maybe there's an extra network request happening somewhere?